### PR TITLE
Add Aaron's slide deck to workshop page

### PR DESCRIPTION
**Pull request by hauten**
Adds PDF to the `workshop22` directory & links to it from `workshop.md`

### CODE REVIEW

**Comment by hauten** [comment]
oh hang on, this intro slide deck is also the wrap-up deck

**Comment by hauten** [comment]
ok, now this is ready

**Comment by hauten** [comment]
Sure, give me a few mins

**Comment by hauten** [comment]
OK @tzanio please review

File: src/workshop.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The meeting activities will take place 7:40am-4:00pm Pacific Daylight Time (GMT-
 | 1:00-2:00 | **Talks, Session IV**<br>(20 mins each)<br><br>Chair: Tzanio Kolev | **Devlin Hayduke** (ReLogic)<br> *Project Minerva: Accelerated Deployment of MFEM Based Solvers in Large Scale Industrial Problems* ([PDF](pdf/workshop22/13_Hayduke_ReLogic.pdf))<br> **Tim Brewer** (Synthetik)<br> *blastFEM: A GPU-accelerated, Very High-performance and Energy-efficient Solver for Highly Compressible Flows* ([PDF](pdf/workshop22/14_Brewer_blastFEM.pdf))<br> **Adolfo Rodriguez** (OpenSim)<br> *Using MFEM for Wellbore Stability Analysis* ([PDF](pdf/workshop22/15_Rodriguez_Wellbore.pdf)) |
 | 2:00-2:20 | **Break** | All |
 | 2:20-2:40 | **MFEM AWS tutorial** | **Julian Andrej** (LLNL) |
```
Comment (tzanio) [inline src/workshop.md:62]: Can we also add a link to the AWS tutorial?

```suggestion
| 2:20-2:40 | **MFEM AWS tutorial** ([Link](../tutorial/)) | **Julian Andrej** (LLNL) |
```

**Comment by hauten** [comment]
Note that the diagram on the tutorial is not rendering properly: https://mfem.org/tutorial/
image

**Comment by tzanio** [comment]
@pazner, can you take a look? Does the diagram render correctly for you?

> Note that the diagram on the tutorial is not rendering properly: https://mfem.org/tutorial/ image

**Comment by pazner** [comment]
Should be fixed by #197

**Comment by tzanio** [comment]
Thanks @pazner !

**Review by tzanio — APPROVED** [review]
Thanks for your help @hauten !

**Pull request merged**